### ISSUE TITLE
make docker-services fails because zauth.h is missing

### ISSUE THREAD

**Issue by her-mes** [wireapp/wire-server#599]
### $ `make docker-services`

```
Step 6/13 : COPY --from=libzauth-builder /usr/include/zauth.h /usr/include/zauth.h
COPY failed: stat /var/lib/docker/overlay2/380620e81dd76b3ea925a64c6b8f3dedf7bf8c612402cefb1e4bb8798e2d1a02/merged/usr/include/zauth.h: no such file or directory
Makefile:124: recipe for target 'docker' failed
make[1]: *** [docker] Error 1
make[1]: Leaving directory '/home/hermes/projects/wire-server/services/nginz'
Makefile:124: recipe for target 'docker-services' failed
make: *** [docker-services] Error 2
```

**Comment by fisx** [comment]
I have reproduced this and will look into it soon.

Thanks for reporting!

**Comment by fisx** [comment]
looks like you (and i) had a stale docker image tagged `local`, and the make rule picked that over the update from quay.io.  when i did this:

```
export DOCKER_TAG=latest
make docker-services
```

i got it to build.  could you please check if that works for you, too?

**Comment by her-mes** [comment]
Hmm.. still not working for me. I tried exporting the environment variable and also tried your pull request. Even after removing all images and trying to `make docker-services` again I'm getting the exact same error.

**Comment by fisx** [comment]
could you compare your output against this?

```
$ docker images | grep quay.io/wire | grep latest
quay.io/wire/alpine-intermediate   latest                                              c62925489329        22 hours ago        1GB
quay.io/wire/zauth                 latest                                              9a1249a37cbc        24 hours ago        55.8MB
quay.io/wire/spar-schema           latest                                              48457d8dddeb        24 hours ago        79.6MB
quay.io/wire/spar-integration      latest                                              8f06e78babcf        24 hours ago        111MB
quay.io/wire/spar                  latest                                              0c4dea4dfa8f        24 hours ago        108MB
quay.io/wire/proxy                 latest                                              a819b493e816        24 hours ago        74.7MB
quay.io/wire/cannon                latest                                              9443b6f193ff        24 hours ago        74.9MB
quay.io/wire/galley-journaler      latest                                              d4f827ef7f02        24 hours ago        90.7MB
quay.io/wire/galley-schema         latest                                              5d30dd8bce62        24 hours ago        77.5MB
quay.io/wire/galley-integration    latest                                              acf079972455        24 hours ago        89MB
quay.io/wire/galley                latest                                              b7f265bb0202        24 hours ago        95.9MB
quay.io/wire/gundeck-schema        latest                                              aa2c43289cfa        24 hours ago        79.6MB
quay.io/wire/gundeck-integration   latest                                              69819718cd7d        24 hours ago        85.3MB
quay.io/wire/gundeck               latest                                              13de20b6c85b        24 hours ago        92.1MB
quay.io/wire/brig-index            latest                                              7d1399fc7bf2        24 hours ago        83.5MB
quay.io/wire/brig-schema           latest                                              d65421ea94e9        24 hours ago        79.6MB
quay.io/wire/brig-integration      latest                                              e60a787d4a7f        24 hours ago        117MB
quay.io/wire/brig                  latest                                              1d2586c80ed9        24 hours ago        118MB
quay.io/wire/alpine-deps           latest                                              c43fe4745ef2        12 days ago         47.4MB
quay.io/wire/alpine-builder        latest                                              378563fa20ee        7 weeks ago         9.21GB
quay.io/wire/alpine-prebuilder     latest                                              65d2e25c72f5        4 months ago        6.16GB
```

and could you also try on #600 ?

**Comment by her-mes** [comment]
I should note that I'm running `make docker-deps docker-intermediate docker-services` when getting this error and I am **not** making `docker-builder` locally. Maybe the issue is with the docker-builder image being pulled from quay.io?
### On latest master branch:
```
$ docker images | grep quay.io/wire | grep latest
quay.io/wire/zauth                 latest              a7d688cf2070        41 minutes ago      55.8MB
quay.io/wire/spar-schema           latest              f687e464908b        41 minutes ago      79.6MB
quay.io/wire/spar-integration      latest              d76af69bc514        41 minutes ago      111MB
quay.io/wire/spar                  latest              5e39471f8c34        41 minutes ago      108MB
quay.io/wire/proxy                 latest              5016f4c84d28        41 minutes ago      74.8MB
quay.io/wire/cannon                latest              80297326627c        41 minutes ago      74.9MB
quay.io/wire/galley-journaler      latest              6ed8f2930f5f        42 minutes ago      90.7MB
quay.io/wire/galley-schema         latest              245c39d72894        42 minutes ago      77.5MB
quay.io/wire/galley-integration    latest              58ab0537d0fd        42 minutes ago      89MB
quay.io/wire/galley                latest              e15608169a59        42 minutes ago      96MB
quay.io/wire/gundeck-schema        latest              3a56360f063d        42 minutes ago      79.6MB
quay.io/wire/gundeck-integration   latest              99c75142268c        42 minutes ago      85.3MB
quay.io/wire/gundeck               latest              9c150729b605        42 minutes ago      92.2MB
quay.io/wire/brig-index            latest              5495ff980a09        42 minutes ago      83.6MB
quay.io/wire/brig-schema           latest              51c740da95e7        42 minutes ago      79.6MB
quay.io/wire/brig-integration      latest              46d5880ada6f        42 minutes ago      117MB
quay.io/wire/brig                  latest              46687107f719        42 minutes ago      118MB
quay.io/wire/alpine-intermediate   latest              a1b61d0ccafe        42 minutes ago      1GB
quay.io/wire/alpine-deps           latest              9f844d0b8151        About an hour ago   47.4MB
quay.io/wire/alpine-builder        latest              f5dac9c01009        2 days ago          9.22GB
```
### On #600 :
```
$ make docker-deps docker-intermediate docker-services
...
Step 6/13 : COPY --from=libzauth-builder /usr/include/zauth.h /usr/include/zauth.h
COPY failed: stat /var/lib/docker/overlay2/40aacb5ce0470ad20794fb206aa41273fa91a4f5d8087d29a12270ac4fa72ce6/merged/usr/include/zauth.h: no such file or directory
Makefile:124: recipe for target 'docker' failed
make[1]: *** [docker] Error 1
make[1]: Leaving directory '/home/hermes/projects/wire-server/services/nginz'
Makefile:124: recipe for target 'docker-services' failed
make: *** [docker-services] Error 2
```
```
$ docker images | grep quay.io/wire
quay.io/wire/zauth                 latest              1efa8b850341        3 minutes ago       55.8MB
quay.io/wire/spar-schema           latest              d193e1622326        4 minutes ago       79.6MB
quay.io/wire/spar-integration      latest              e663649d29b5        4 minutes ago       111MB
quay.io/wire/spar                  latest              4d20365c3080        4 minutes ago       108MB
quay.io/wire/proxy                 latest              7795cd015c75        4 minutes ago       74.8MB
quay.io/wire/cannon                latest              d6f520a8640d        4 minutes ago       74.9MB
quay.io/wire/galley-journaler      latest              1a4a1ce0b6d8        4 minutes ago       90.7MB
quay.io/wire/galley-schema         latest              108062799b17        4 minutes ago       77.5MB
quay.io/wire/galley-integration    latest              019bbf7a04fe        4 minutes ago       89MB
quay.io/wire/galley                latest              2b717ab406da        4 minutes ago       96MB
quay.io/wire/gundeck-schema        latest              68aa1b461a80        4 minutes ago       79.6MB
quay.io/wire/gundeck-integration   latest              a77aa0e3d094        4 minutes ago       85.3MB
quay.io/wire/gundeck               latest              616114a062d3        4 minutes ago       92.2MB
quay.io/wire/brig-index            latest              30827d5923d9        4 minutes ago       83.6MB
quay.io/wire/brig-schema           latest              05bc36f32b58        4 minutes ago       79.6MB
quay.io/wire/brig-integration      latest              59eb89a61a6b        4 minutes ago       117MB
quay.io/wire/brig                  latest              25e8cb2a9b38        4 minutes ago       118MB
quay.io/wire/alpine-intermediate   latest              6ca3e0e44d80        4 minutes ago       1GB
quay.io/wire/alpine-deps           latest              fec2aea92c30        17 minutes ago      47.4MB
quay.io/wire/alpine-builder        latest              f5dac9c01009        2 days ago          9.22GB
```

**Comment by jschaul** [comment]
`cd services/nginz && make docker` is sufficient to reproduce this issue (it is exclusively related to nginz, not any other services, as the error message indicates). The docker image building fails since [this commit](https://github.com/wireapp/wire-server/commit/d30ec9a4319392b8d38ec2643f303635cc5c451b) due to a rename of the `PREFIX` variable. 
A fix for this is in #605.
